Gemfile

```diff
@@ -26,7 +26,7 @@ gem "jsonapi-serializer"
 # gem "bcrypt", "~> 3.1.7"
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem "tzinfo-data", platforms: %i[ windows jruby ]
+# gem "tzinfo-data", platforms: %i[ windows jruby ]
```

Comment:
I'm not sure about merging this to the dev with some stuff in the gemfile commented out. I don't know if this will cause any issues in the dev branch.

Reply:
When we are tackling the merge conflicts, we can uncomment that out.  This was a relic from my struggles early thhis morning that I forgot to re-comment